deepchecks/nlp/utils/text_properties.py

```diff
@@ -263,6 +330,8 @@ def calculate_default_properties(
         ignored, even if they are in the include_properties parameter.
     device : int, default None
         The device to use for the calculation. If None, the default device will be used.
+    models_storage : Union[str, pathlib.Path, None], default None
+        a directory to store the models.
```

Comment:
Add desc on where it is saved when None

Reply:
Add info that if the folder already contains the relevant resources they are not re-downloaded (+ verify it happens :) )

Reply:
added